app/src/main/java/org/simple/clinic/selectcountry/Country.kt

```diff
@@ -0,0 +1,10 @@
+package org.simple.clinic.selectcountry
+
+import java.net.URL
```

Comment:
Not sure with the `java.net.URL`. Was this intentional?

Reply:
https://stackoverflow.com/a/3771123/421372

Reply:
Using a type instead of a `String` was intentional. This is to expose the intention behind the property instead of using a primitive.

I didn't know about the problem with the `java.net.URL` type, will change to `java.net.URI` instead!

Reply:
Can we use our own type instead? Using a `URI` or `URL` means, it is possible to initiate a side-effect from `Init` or `Update`.

Reply:
This makes sense. @rakshakhegde and I will make this change in the next PR?

Reply:
Works